src/controllers/ReactComponentController.tsx

```diff
@@ -25,6 +26,7 @@ const ReactComponentController = ({
       <StimulusComponent
         parameters={parameters}
         trialId={trialId}
+        setAnswer={setAnswer}
```

Comment:
This is the location of the API for the users' stimulus components. Nothing todo here, just making a note for us